ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/ConflictCheckWithCommittedImpl.scala

```diff
@@ -149,4 +158,96 @@ private[validate] class ConflictCheckWithCommittedImpl(
     }
   }
 
+  private def sameContractData(
+      actual: (Value.VersionedContractInstance, Timestamp),
+      provided: DisclosedContract,
+  )(implicit loggingContext: LoggingContext): Boolean = {
+    // TODO DPP-1026: Note: Not validating transaction version or agreement text here, because we're making those up
+
+    // TODO DPP-1026: Template id validation can probably be removed once we switch from VersionedContractInstance to Value
+    val actualTemplate = actual._1.unversioned.template
+    val providedTemplate = provided.contractInst.unversioned.template
+
+    // TODO DPP-1026: Do we need the user to provide verbose arguments?
```

Comment:
> TODO DPP-1026: Do we need the user to provide verbose arguments? 

We have two solution: 
1. Either we require the contract payload to be normalized 
2.  Or we are as lenient that for the content of the argument (there is more to take into account than just the type info, there is for instance the record field order, the map ordering ... ).

In my opinion, we should go with 1 for the PoC. For the final implementation we should  require the value to be normalized.